rosco-web/config/rosco.yml

```diff
@@ -191,6 +191,34 @@ google:
         virtualizationSettings:
           sourceImage: ubuntu-1404-trusty-v20150909a
 
+openstack:
+  enabled: ${OS_ENABLED:false}
+  bakeryDefaults:
+    identityEndpoint: ${OS_AUTH_URL:https://openstack.yourcompany.com:5000/v3}
+    domainName: Default
+    floatingIpPool: ext_vlan1472_net
+    securityGroups: default
+    tenantName: PRD-spinnakeropenstack
```

Comment:
Sorry, I was just asking if `PRD-spinnakeropenstack` was a special value that you intended to commit. Or a placeholder, or something else.